packages/manager/apps/pci-savings-plan/src/components/Dashboard/Kpis/Kpis.tsx

```diff
@@ -49,53 +53,60 @@ const Kpi = ({
         </OdsTooltip>
       </OdsText>
       <span className="text-[20px] font-bold mb-0 text-[#0050D7]">{value}</span>
+      {valueSavedWithoutAmount && (
+        <div className="flex items-center">
+          <span className="text-[15px] mb-0 text-[#666ca1] mr-2">
+            {t('dashboard_kpis_saved_strikethrough')}
+          </span>
+          <span className="line-through text-[15px] mb-0 text-[#666ca1]">
+            {valueSavedWithoutAmount}
+          </span>
```

Comment:
Price should be integrated inside the translation to be able to move it where it belongs in the sentence

I think we should also consider putting the other price inside to make a full sentence